float32 theta
float32 distance
float32 x1
float32 y1
float32 orientation
int32 new_simulation
---
float32 distance
float32 theta